deploy/platform/kubernetes/feature-cluster/resources.yaml

```diff
@@ -20,7 +20,7 @@
 apiVersion: v1
 data:
   metadata-service-mapping.yaml: |
-    serviceName: agentless::${LABELS."service.istio.io/canonical-name",LABELS."app.kubernetes.io/name",LABELS.app}
```

Comment:
Just change these all, no need to keep `agentless` for old feature

Reply:
updated, please recheck